packages/client/hmi-client/src/page/Home.vue

```diff
@@ -376,7 +371,7 @@ header > section > button {
 	padding-left: 0.5rem;
 }
 
-.p-inputtext {
+.tera-input {
```

Comment:
where is this class used?

Reply:
thats the class used for all `tera-inputs-text`/`tera-input-number` components. This just adds that min-width to that tera input rather than the old primevue input